tests/Core/RouterTest.php

```diff
@@ -48,6 +48,19 @@ public function test_catch_exception_when_routes_file_is_in_wrong_format()
         $this->router->get();
     }
 
+    public function test_no_exceptions_happen_with_correct_url()
+    {
+        $this->expectNotToPerformAssertions();
+        $routes = [
+            'GET' => [
+                '/ping' => 'HealthCheckController@ping',
+            ],
+        ];
+        $this->router = new Router($routes, $this->request);
+        $this->get('/ping');
+        $this->router->get();
```

Comment:
Where is the new line before last call :laughing:

Reply:
You don't have it in all other test cases, so I tried to follow 😛 

But to be serious, it is not clear what is the main action of test method. Normally, it is some kind of `assert*` but in this particular test class, we set expectations about exceptions and as a result, no assertions needed. So it is fine to keep it in one block in the end.